themes/veg/_index.scss

```diff
@@ -0,0 +1,84 @@
+$font-stack: 'Barlow';
+
+body {
+  font-family: $font-stack;
+  background-image: url("/assets/background.jpg");
```

Comment:
This needs to use `{{assetPath}}//background.jpg`, and moved to `index.hbs` (because that's where the templating is done.

So remove this line.

And in `index.hbs` add a `<style>` tag after this one is loaded, and override `body { background-image: url("{{assetPath}}/background.jpg")}`.

Reply:
Done